### PR TITLE
Add release context, to help it update Github?

### PR DESCRIPTION
**Pull request by sbrandwoo**
Release build is failing when trying to push release information to GitHub: https://circleci.com/gh/transferwise/release-to-github-with-changelog/111

Maybe it needs this context?

### CODE REVIEW

**Review by gusso — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -37,6 +37,7 @@ workflows:
     jobs:
       - test
       - release:
+          context: frontend-deploy-library
```
Comment (oliverviljamaa) [inline .circleci/config.yml]: This looks like the context we use for private CircleCI (see [pay-in](https://github.com/transferwise/pay-in/blob/4fa66eb76ea5498015f2ade410d8a38c7aa74c83/.circleci/config.yml#L101)), public repos have this (see [public-navigation](https://github.com/transferwise/public-navigation/blob/c1a732d9995798a679c3806978a1343a44780beb/.circleci/config.yml#L132)):
```suggestion
          context: frontend-publish
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# v1.2.4
+
+## Fix build
```
Comment (oliverviljamaa) [inline CHANGELOG.md:3]: The previous version didn't get released, so no need to bump — it's not changing the lib, right?

Comment (gusso) [reply CHANGELOG.md:3]: It did: https://www.npmjs.com/package/release-to-github-with-changelog

File: .circleci/config.yml (hunk 1 above)
Comment (sbrandwoo) [reply .circleci/config.yml]: this is a dangerous naming convention

File: CHANGELOG.md (hunk 2 above)
Comment (sbrandwoo) [reply CHANGELOG.md:3]: Indeed, only the github changelog bit broke, so we need to repush a new version too :(

Comment (oliverviljamaa) [reply CHANGELOG.md:3]: Oh crap!

**Review by oliverviljamaa — APPROVED** [review]

**Pull request merged**